### ISSUE TITLE
Layout: LayoutPane not respecting XHTML attributes

### ISSUE THREAD

**Issue by aleksei-opritov** [primefaces-extensions/primefaces-extensions#155]
Seems that some attributes (closable, minSize, maxSize) for pe:layoutPane are not working if  a layout is configured by options defined as attributes of layoutPane component. If a layoutPane is created  via LayoutOptions as Java model it works.

**Environment:**
 - PF Version: 8.0.3
   PF extension Version 8.0.4
 - JSF + version:2.3
 - Affected browsers:  Chrome, Firefox

**Expected behavior**
It would be nice to also create a layout by options defined as attributes of pe:layoutPane. 

**Sample XHTML**
```xml
 <pe:layout id="documentLayout" fullPage="false"
                                        stateCookie="false"
                                        state="#{documentLayoutController.documentLayoutState}"
                                        widgetVar="documentLayoutWidget">
                                <p:ajax event="resize" process="@this" update="@none"/>
                                <!--West pane-->
                                <pe:layoutPane id="westCnt"
                                               position="west"
                                               closable="false"
                                               size="#{documentLayoutController.westSize}"
                                               minSize="20%"
                                               maxSize="70%"
                                               rendered="#{documentLayoutController.layoutState.west}">
                                    <p:outputPanel layout="block" id="westPane">
                                        <h:form>
```

**Comment by melloware** [comment]
I have verified your issue thanks for reporting.  Fixed for 9.0.

**Comment by sasynkamil** [comment]
Just note, this is working in PF Extension 8.0, later versions are broken.

**Comment by GedMarc** [comment]
@sasynkamil can you explain a bit more please? later as in version 9, or later as in version 8.0.x

**Comment by melloware** [comment]
Yep it worked in 8.0 and was broken in 8.0.1 and its fixed for 9.0

**Comment by StevenHachel** [comment]
please fix it for version 8 as well. 😁
